src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -92,6 +99,7 @@ def get_attrs(
     ) -> defaultdict[AlertRule, Any]:
         alert_rules = {item.id: item for item in item_list}
         prefetch_related_objects(item_list, "snuba_query__environment")
+        prefetch_related_objects(item_list, "activations")
```

Comment:
Why do we add this? It'll fetch all activations for the rules into memory

Reply:
Oof
this is left over from when i was trying to figure out how to get this work.

will remove